packages/graphback-core/src/scalars/scalars.ts

```diff
@@ -1,12 +1,34 @@
 import { GraphQLNamedType, GraphQLScalarType } from 'graphql';
-import { GraphQLJSON, GraphQLJSONObject } from 'graphql-compose';
+import { JSONResolver, JSONObjectResolver } from 'graphql-scalars';
 
-const graphbackScalarTypes: GraphQLScalarType[] = [GraphQLJSON, GraphQLJSONObject]
+/**
+ * GraphbackJSON scalar
+ *
+ * Wraps Urigo/graphql-scalars JSON
+ */
+export const GraphbackJSON = new GraphQLScalarType({
+  ...JSONResolver.toConfig(),
+  name: 'GraphbackJSON'
+  // TODO: Custom description
+})
+
+/**
+ * GraphbackJSONObject scalar
+ *
+ * Wraps Urigo/graphql-scalars JSONObject
+ */
+export const GraphbackJSONObject = new GraphQLScalarType({
+  ...JSONObjectResolver.toConfig(),
+  name: 'GraphbackJSONObject'
+  // TODO: Custom description
+})
+
+const graphbackScalarTypes: GraphQLScalarType[] = [GraphbackJSON, GraphbackJSONObject]
 
 /**
  * Checks if the type is on the the default Graphback supported scalars
  *
- * @param type - GraphQL type
+ * @param type - any type
  */
 export function isSpecifiedGraphbackScalarType(type: GraphQLNamedType): boolean {
   return graphbackScalarTypes.some(({ name }: GraphQLScalarType) => type.name === name);
```

Comment:
If we had 
```
const graphbackScalarTypes = {
       GraphbackJSON: true,
       GraphbackJSONObject: true
}
```

This check could be re-written to 
```
return graphbackScalarTypes[type.name];
```

Reply:
True, but then you have to maintain two lists referencing the Graphback scalars (the actually scalars + `graphbackScalarTypes`.

I copied the same function from graphql-js:

https://github.com/graphql/graphql-js/blob/58aaa1df30b501b67d6986ea2fc1a8b95ac9fbca/src/type/scalars.js#L286

Reply:
> True, but then you have to maintain two lists referencing the Graphback scalars (the actually scalars + `graphbackScalarTypes`.
> 
> I copied the same function from graphql-js:
> 
> https://github.com/graphql/graphql-js/blob/58aaa1df30b501b67d6986ea2fc1a8b95ac9fbca/src/type/scalars.js#L286

I might have missed something. Why maintain two lists?

Reply:
Maybe not two "lists" but you would no longer have a single source of truth. If the actual scalar gets renamed to GBJSON you will also need to update the key `graphbackScalarTypes.GraphbackJSON` to match it.

Whereas with the current implementation, The list is created from the scalar objects and referenced in this function.

https://github.com/aerogear/graphback/blob/json-scalar/packages/graphback-core/src/scalars/scalars.ts#L9-L26

Reply:
> Maybe not two "lists" but you would no longer have a single source of truth. If the actual scalar gets renamed to GBJSON you will also need to update the key `graphbackScalarTypes.GraphbackJSON` to match it.

I see what you mean, I guess this would limit such an issue
```js
const graphbackScalarTypes = {
       [GraphbackJSON.name]: true,
       [GraphbackJSONObject.name]: true
}
```

It is a nitpick. I've no strong opinion on this.